src/executables/util_cs.c

```diff
@@ -3212,6 +3216,10 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
 
   check_replica_info = (HA_GET_MODE () == HA_MODE_REPLICA);
   pageid = utility_get_option_bigint_value (arg_map, APPLYINFO_PAGE_S);
+  if ((pageid != NULL_LOG_PAGEID) && (log_path == NULL))
```

Comment:
It is more readable to use the macro, but I think you should change the default value of the parameter along, which is -1.